src/swaggerHandler.js

```diff
@@ -105,6 +105,11 @@ module.exports = curry((schema, options, req, res, next) => {
     return next(new HttpError.InternalServerError('No swagger schema defined'));
   }
 
+  if (options.cors) {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Headers", "X-Requested-With");
+  }
```

Comment:
👍 assuming wide open is what we want. maybe we can accept a whitelist param here later